launcher/src/main/java/io/snappydata/tools/QuickLauncher.java

```diff
@@ -124,6 +124,10 @@ private int start(final String[] args) throws IOException, InterruptedException
     // add java path to command-line first
     commandLine.add(System.getProperty("java.home") + "/bin/java");
     commandLine.add("-server");
+    // https://github.com/airlift/jvmkill is added to libgemfirexd.so
+    // adding agentpath helps kill jvm in case of OOM. kill -9 is not
+    // used as it fails in certain cases
+    commandLine.add("-agentpath:" + snappyHome + "/jars/libgemfirexd.so");
```

Comment:
Shouldn't this use either libgemfirexd64.so or libgemfirexd.so depending on architecture? You can check for it using SharedLibrary.is64Bit() or Native.is64Bit().